src/main/java/pvpmode/internal/server/PvPServerEventHandler.java

```diff
@@ -625,12 +636,27 @@ public void onPlayerLoggedIn (PlayerLoggedInEvent event)
                 messagePart0.appendSibling (messagePart1.appendSibling (messagePart2).appendSibling (messagePart3)
                     .appendSibling (messagePart4).appendSibling (messagePart5)));
         }
+
+        server.getPvPDataManager ().onLogin (event.player);
+
     }
 
     @SubscribeEvent
     public void onPlayerLoggedOut (PlayerLoggedOutEvent event)
     {
         server.getClientsideSupportHandler ().removeSupportedClient ((EntityPlayerMP) event.player);
+
+        server.getPvPDataManager ().onLogout (event.player);
+    }
+
+    @SubscribeEvent
+    public void onWorldTick (WorldTickEvent event)
+    {
+        if (event.phase == Phase.END && event.world.getWorldTime () % 1200 == 0) // TODO Currently hardcoded, make
+                                                                                 // configurable with Lembas
```

Comment:
Make this look a bit nicer
```suggestion
        // TODO Currently hardcoded, make configurable with Lembas
        if (event.phase == Phase.END && event.world.getWorldTime () % 1200 == 0)
```